docs/services/cosmosdb/cosmosdb.md

```diff
@@ -0,0 +1 @@
+[CRD](charts/azure-service-operator/crds/apiextensions.k8s.io_v1beta1_customresourcedefinition_cosmosdbs.azure.microsoft.com.yaml)
```

Comment:
This is kinda a weird document. It just has a single link without any headings or anything?

Reply:
Just added a bit more info, but this could def use more love.